.github/workflows/ci.yml

```
@@ -90,13 +91,41 @@ jobs:
         with:
           components: rustfmt, clippy
 
-      - name: Install pgvector and pg_trgm into template1
+      - name: Build yorishiro binary
+        run: cargo build --locked -p yorishiro --bin yorishiro
+
+      - name: Install sea-orm-cli for entities regen
+        run: cargo install sea-orm-cli --locked --features postgres-vector --version $(grep 'sea-orm = { version' Cargo.toml | head -1 | sed 's/.*version = "//;s/".*//' | tr -d '=')
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:73: Quote this to prevent word splitting

(shellcheck)